subiquity/common/types.py

```diff
@@ -356,21 +356,21 @@ class ShutdownMode(enum.Enum):
 
 @attr.s(auto_attribs=True)
 class WSLConfigurationBase:
-    custom_path: str = attr.ib(default='/mnt/')
-    custom_mount_opt: str = ''
-    gen_host: bool = attr.ib(default=True)
-    gen_resolvconf: bool = attr.ib(default=True)
+    automount_root: str = attr.ib(default='/mnt/')
```

Comment:
I already mentioned it as a nitpick, but you didn’t answer and keep /mnt/ instead of just /mnt.

Reply:
The default value for this option by Microsoft is `/mnt/` though

![image](https://user-images.githubusercontent.com/15316889/135947727-27b2e264-8b05-4eca-8072-a08ede9190b7.png)

Reply:
Ack, we should discuss that with Microsoft, as it’s a nitpick but this isn’t the best practice to put in fstab for instance.